indy_client/test/cli/test_pool_upgrade_schedule.py

```diff
@@ -0,0 +1,26 @@
+import dateutil
```

Comment:
I think this test needs to be in indy_node.test.upgrade

Reply:
This test operates on CLI level (depends on `poolUpgradeSubmitted` fixture). So it should be located in `indy_client` package.